ONSdigital/eq-author-app
# Use new content picker for destination

### What is the context of this PR?
This pr changes the content picker to v2 when picking a routing destination.

### How to review 
Check that the new content picker follows the designs and still does what it should do. 

Steps to test:
1. Create a new questionnaire with multiple sections and multiple questions in each section.
1. Try and change the routing destination and make sure it follows the rules below.
1. Try and change the else destination.
1. Try and add multiple rules and change their destination.
1. Resize window to see that it fits different screen sizes.

**It should follow the following rules:** 
* Only questions should be displayed in the picker.
* Only questions **after** the current question should be displayed.
* In sections after the current section, only the first question should be displayed in the picker. 
* Selecting another sections question, routes to the section not the question.
